db/migrate/20191009171351_mosquito_genome_update_2019_10_02.rb

```diff
@@ -0,0 +1,9 @@
+class MosquitoGenomeUpdate20191002 < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
is this actually reversible? rails must be really magic then!

if not then change the method to `up`

Reply:
Nope, thanks for clearing that up.